analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SupplyParameterFromQueryNeedRoutableComponent_Compliant.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+[Route("/query-parameters")]
+public class SupplyParameterFromQueryNeedRoutableComponent_Compliant : ComponentBase
```

Comment:
In general, there are in principle many other possible scenarios of interest: "Attribute" suffix, attribute parameters, aliases for the parameters, custom attributes named as the ASPNET ones etc. 

However, knowing that the implementation always uses `HasAttribute`, which takes advantage of the semantic level, I don't think we need to go that way.